docs/introduction.rst

```diff
@@ -22,22 +29,17 @@ organization based on location, industry sector, and deployed platforms.
 Framing our Analysis
 --------------------
 
-Adversaries are constantly evolving their attacks, driving up the cost of intrusions.
-Consequently, defenders must continue to protect against an increasing amount of
-adversary techniques and behaviors. Despite their best efforts, it is not possible to
-defend against all potential scenarios. This raises the questions, “How many MITRE
-ATT&CK techniques apply to the average organization?” and “Which techniques does an
-organization realistically need to defend against?”
+While our volume of data has increased significantly, there are caveats to keep in mind when reading our results. Primarily, we are limited to the data we were provided. While getting the data straight from vendors is beneficial, it introduces biases. To ameliorate this bias, we have a diverse set of providers and a large data set, which reduces some skewing within the data. 
```

Comment:
Can you please word wrap the text? I can show you how to set this up in Visual Studio Code if you don't have it already.